### DISCUSSION TITLE
who knows that does assimp support ZTL format?

### DISCUSSION THREAD

**Discussion by gordenfl** [assimp/assimp#5115] [General]
Does assimp support ZTL format?

**Comment by NicusorN5** [comment]
Here's the list of supported formats: https://assimp-docs.readthedocs.io/en/latest/about/introduction.html

And as far as I can see, ZTL is not supported.

**Comment by tellypresence** [comment]
Doesn't seem to be a lot of information out there; appears to be a proprietary, closed binary format so not likely would be able to support this format

#### References
- [Information parsing on ZTL files](https://www.zbrushcentral.com/t/information-parsing-on-ztl-files/341763)
   - "ZTL is a closed binary format" 
- [convert zbrush file?](https://blenderartists.org/t/convert-zbrush-file/472099)
   - "only zbrush can open ztl files"
